README.md

```diff
@@ -46,6 +46,15 @@ Start the server
   npm run start
 ```
 
+Start with PM2 ~ Tested with Ubuntu 20 server
```

Comment:
Cocoknya diganti "Or Start as Background Process with PM2: Process Manager (For Production)"

Kata OR dibutuhkan karena dapat memilih antara opsi diatasnya atau opsi ini

dan diberikan label "Optional" serta keterangan jika step ini hanya dibutuhkan jika butuh menjalankan pada production dan butuh berjalan pada backround process

Reply:
sudah di up pak